### PR TITLE
Mint preview TypeScript refactor

### PR DESCRIPTION
**Pull request by BrandonC123**
Task: 
- remove // @ts-nocheck
- and fix ts errors in mintPreview.tsx

### CODE REVIEW

File: components/mintPreview.tsx [hunk 1]
```diff
@@ -21,25 +33,14 @@ function MintPreview({ hash }) {
           console.log("error.response");
         }
       });
-  }, [hash]);
-
+  }
   return (
     <div>
-      <img src={image} style={{ width: "100%" }}></img>
+      <Image src={image} width="0" height="0" sizes="20vw" className="w-full h-auto" alt="" />
```
Comment (tanerdurmaz) [inline components/mintPreview.tsx]: we want to keep image ratio

**Review by tanerdurmaz — CHANGES_REQUESTED** [review]
need to keep aspect ratio of iamges

**Comment by BrandonC123** [comment]
Reverted Image to img tag and also added back in missing tailwind code for referral component

**Review by tanerdurmaz — APPROVED** [review]

**Comment by tanerdurmaz** [comment]
when chance can you share this in general channel one sentence is enough like
"Mint preview typescript refactor" and add link to the this pr
now on devs will share each update

**Pull request merged**